package.json

```diff
@@ -203,9 +203,37 @@
   },
   "devDependencies": {
     "@babel/core": "7.0.0-beta.56",
+    "@babel/generator": "7.0.0-beta.56",
```

Comment:
for all the `beta`, I think we want to have babel 7.0 support ASAP so I think we should not merge this until we have everything on `7.0`.

Reply:
Agree, should be switching over to the release version of v7 - I had to add these in at a fixed version to align with metro's version of `babel/core` - once metro has been updated to use Babel 7 then all of these except for `@babel/core` can be removed.

`metro-react-native-babel-preset`'s Babel packages would also need updating to the same version of Babel as used here and in `metro` - to avoid random bugs/issues.

Reply:
yup, there is already a PR for that :)

Reply:
Ooo noice, missed that one - ok is it worth merging this on `beta.56` (after I sort below changes though) to get build/CI working as is - then once that metro PR is landed and published to NPM I can drop another PR here to update it to the latest versions.

Or we can just wait for metro to be updated - is another option 🙃

Reply:
TBH since the PR is passing all the tests  I'd prefer to first have the Metro team release a new version and synch up RN master https://github.com/facebook/metro/pull/233